UI/mainview.cpp

```diff
@@ -119,6 +119,21 @@ void MainView::LoadSettings() {
                    ui->cmbFontSize->currentText().toInt());
 }
 
+void MainView::SetupShortuctKeys() {

```

Comment:
I wonder if we need to delete these

Reply:
Since we are not creating them again and again we won't be leaking though. Any memory left over like this will be claimed by system later on anyway. So It's not harmful.